elements/includes/creative_commons.inc

```diff
@@ -15,14 +23,27 @@
  *   The element definition.
  */
 function xml_form_elements_creative_commons($element, &$form_state) {
-  $creative_commons_base_url = "http://creativecommons.org/licenses/";
+  $license_options = array(
+    'none' => t('Do not output any license'),
```

Comment:
@DiegoPino I was thinking to this, initially, so that the choice is always up-to-date. But:
- do we also want to support the deprecated 'sample+' license? (we could nuke it, though)
- new licenses might break other logic (in particular checks)
- need default set, anyway, when API is unreachable
- I wanted to reduce API requests

I, then, decided against it, but I would be happy to implement this retrieval, if that is what is wanted.